<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QLabel" name="image1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1061</width>
      <height>781</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>692</width>
      <height>0</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>980</y>
      <width>1920</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgba(0, 0, 0, 100)</string>
    </property>
    <widget class="QLabel" name="song_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>300</width>
       <height>16</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: none;</string>
     </property>
     <property name="text">
      <string>A Day to Remember</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="album_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>300</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: none;</string>
     </property>
     <property name="text">
      <string>Bad Vibrations</string>
     </property>
    </widget>
    <widget class="QLabel" name="artist_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>300</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: none;</string>
     </property>
     <property name="text">
      <string>Darkest days</string>
     </property>
    </widget>
    <widget class="QLabel" name="release_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>300</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: none;</string>
     </property>
     <property name="text">
      <string>2016-07-2019</string>
     </property>
    </widget>
    <widget class="QLabel" name="song_length">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color: none;</string>
     </property>
     <property name="text">
      <string>4:20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_account"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="menu_account">
   <property name="text">
    <string>Account</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
